Assets/QuestionGenerator.cs

```diff
@@ -38,7 +38,7 @@ void Start()
             wrongAnswer2 = Random.Range(answer-10, answer+10);
             wrongAnswer3 = Random.Range(answer-10, answer+10);
         } while (wrongAnswer1 == answer || wrongAnswer2 == answer || wrongAnswer3 == answer 
-            && wrongAnswer1 == wrongAnswer2 || wrongAnswer1 == wrongAnswer3 || wrongAnswer2 == wrongAnswer3);
+            || wrongAnswer1 == wrongAnswer2 || wrongAnswer1 == wrongAnswer3 || wrongAnswer2 == wrongAnswer3);
```

Comment:
Instead of generating all three wrong answers every loop, could we do smth like this:
```
wrongAnswer = new List<int>()
for 0 to 2 {
  do {
    rand = random(ans-10, ans+10)
  } while rand in wrongAnswer || rand == ans
  wrongAnswer.add(rand)
}
```

Reply:
Fixed anyway xd